### PR TITLE
Idempiere 4688

### PR DESCRIPTION
**Pull request by mschnbeck**
Added CustomsTariffNumber to m_product and i_product, ImportProduct.java imports this now. Also group1 and group2 added to i_product and the import in ImportProduct.java

### CODE REVIEW

File: org.adempiere.base.process/src/org/compiere/process/ImportProduct.java [hunk 1]
```diff
@@ -559,10 +562,12 @@ protected String doIt() throws java.lang.Exception
 						.append("SET (Value,Name,Description,DocumentNote,Help,")
 						.append("UPC,SKU,C_UOM_ID,M_Product_Category_ID,Classification,ProductType,")
 						.append("Volume,Weight,ShelfWidth,ShelfHeight,ShelfDepth,UnitsPerPallet,")
+						.append("CustomsTariffNumber,Group1,Group2,")
```
Comment (CarlosRuiz-globalqss) [inline org.adempiere.base.process/src/org/compiere/process/ImportProduct.java:565]: This is just the "update" case, you must take care also in the "insert" case in line 541:
`MProduct product = new MProduct(imp);`
The constructor MProduct(X_I_Product) must also assign these three new columns.

Also - as you added columns, please generate the classes X_/I_ for M_Product and I_Product.

Comment (mschnbeck) [reply org.adempiere.base.process/src/org/compiere/process/ImportProduct.java:565]: Done. To do the first model classes where required anyway.

**Review by CarlosRuiz-globalqss — APPROVED** [review]
Hi @mschnbeck - this is ready for merging.

Tested successfully.

The only pending thing is documentation in the wiki new features.

Regards,

Carlos Ruiz

**Comment by mschnbeck** [comment]
Hi @CarlosRuiz-globalqss,
NF-entry created, but I'm not sure, whether to sort it under 8.2 or 9.1

**Pull request merged**